src/components/panel/panel.js

```diff
@@ -1104,16 +1102,19 @@ MdPanelRef.prototype._createPanel = function() {
                 getElement(self._config['attachTo']));
           }
 
-          self._addStyles();
           self._configureTrapFocus();
-          return resolve(self);
+          return self._addStyles().then(function() {
+            self._panelContainer.addClass(MD_PANEL_HIDDEN);
+            return resolve(self);
```

Comment:
Do you still need to return the resolve(self)? That feels like it is mixing 2 styles.